components/merkledb/src/backends/temporarydb.rs

```diff
@@ -87,11 +97,23 @@ impl TemporaryDB {
             .write_opt(batch, &write_options)
             .map_err(Into::into)
     }
+
+    fn temporary_snapshot(&self) -> TemporarySnapshot {
+        TemporarySnapshot {
+            snapshot: self.inner.rocksdb_snapshot(),
+            _dir: Arc::clone(&self.dir),
+        }
+    }
+
+    #[cfg(test)]
+    fn get_db_path(&self) -> &Path {
```

Comment:
Nit: IMO, this method could be inlined; adding a separate one-line method just for tests looks like an overkill.